test/utils/execution.js

```diff
@@ -57,6 +57,26 @@ let executeTransactionWithSigner = async function(signer, safe, subject, account
     let baseGasEstimate = estimateBaseGas(safe, to, value, data, operation, txGasEstimate, txGasToken, refundReceiver, accounts.length, nonce) + extraGas
     console.log("    Base Gas estimate: " + baseGasEstimate)
 
+    let additionalGas = 10000
+    for (let i = 0; i < 100; i++) {
+        try {
+            let estimateData = safe.contract.requiredTxGas.getData(to, value, data, operation)
```

Comment:
This coud be removed and the value from the previous estimateData (line 46) could be used